cloud_governance/common/clouds/aws/utils/common_methods.py

```diff
@@ -26,3 +31,21 @@ def get_tag_value_from_tags(tags: list, tag_name: str, cast_type: str = 'str',
                         return str(tag.get('Value').strip())
                 return tag.get('Value').strip()
     return default_value
+
+
+def boto3_client(client: str, region_name: str, **kwargs):
```

Comment:
I dont know if it good idea to create method only for aws boto3 client initialization

Reply:
Instead of calling the boto3 method every time, I thought to create one method to return the client.

Reply:
It slight confused for code reviewer to see that calling the client with boto3_client instead of boto3.client

Reply:
I intend to make better exception handling of client errors. 
May we can change the method name?

Reply:
leave it as is

Reply:
Did u replace all the place that using boto3 client ?

Reply:
No, I will open the new PR.

Reply:
ok